Airports/EGVA/Positions.txt

```diff
@@ -0,0 +1,4 @@
+EGVA_D_APP:Fairford Director:134.550:VAD:D:EGVA:APP:-:-:3740:3745:N051.41.00:W001.47.00::
+EGVA_TWR:Fairford Tower:124.800:VAT:A:EGVA:TWR:-:-:3740:3745:N051.41.00:W001.47.00::
+EGVA_GND:Fairford Ground:121.170:VAG:A:EGVA:GND:-:-:3740:3745:N051.41.00:W001.47.00::
+EGVA_ATIS:Fairford ATIS:135.450:VAI:A:EGVA:ATIS:-:-::::::
```

Comment:
no VHF ATIS at VA.

Reply:
Source for this frequency:

U3582/20 NOTAMN
Q) EGTT/QCACF/IV/B  /A /000/999/5141N00147W005
A) EGVA B) 2008061258 C) 2011012359
E) RAF FAIRFORD NEW FREQUENCIES:
GROUND CONTROL 339.7MHZ
TOWER 246.35MHZ 
UHF ATIS 292.3MHZ
VHF ATIS 135.455MHZ
SUPERVISOR OF FLIGHT 279.275MHZ
ALL OTHER PUBLISHED FREQUENCIES REMAIN UNCHANGED.
CREATED: 06 Aug 2020 13:05:00 
SOURCE: EUECYIYN

If you want to ignore it then I'll get it removed

Reply:
Right ok i am assuming they will be included in the next cycle if they are updating the freqs. As for the other freqs I think they are only used during RIAT and notam'd when in use, I don't see the point in having them for an airfield that is barely used.